openlibrary/templates/lib/header_dropdown.html

```diff
@@ -12,25 +12,52 @@
 <div class="$(props['name'])-component header-dropdown">
     <details>
       <summary>
-        $if 'image' in props:
+        $if 'mask' in props and ctx.user:
```

Comment:
Why does this check `if mask`?

Reply:
to check for hamburger prop will change it to `props['name'] = 'hamburger'`